src/locales/strings/enUS.json

```diff
@@ -623,12 +623,18 @@
   "transaction_details_exchange_support_request": "%s Support Request",
   "transaction_details_fee_warning": "High Network Fees",
   "transaction_details_swap": "Swap Funds",
+  "transaction_details_swap_network_fee": "Swap Network Fee",
   "transaction_details_swap_order_cancel": "Swap Order Canceled",
   "transaction_details_swap_order_post": "Swap Order Opened",
   "transaction_details_swap_order_fill": "Swap Order Filled",
+  "transaction_details_claim": "Claim Staked Funds",
+  "transaction_details_claim_order": "Claim Order",
   "transaction_details_stake": "Stake Funds",
   "transaction_details_stake_order": "Stake Order",
+  "transaction_details_stake_network_fee": "Stake Network Fee",
+  "transaction_details_transfer_network_fee": "Transfer Network Fee",
   "transaction_details_unstake": "Unstake Funds",
+  "transaction_details_unstake_network_fee": "Unstake Network Fee",
```

Comment:
We can probably safely do `"%$1s Network Fee"` for these

Reply:
Technically the translations might change the order of the words so I would say best not to